test/unit/models/instance_spec.js

```diff
@@ -0,0 +1,155 @@
+"use strict";
+
+var Instance = require("../../../lib/models/Instance");
+var _        = require("lodash");
+var expect   = require("chai").expect;
+
+describe("The services schema", function () {
```

Comment:
This is a nice test suite. However, it would be worth considering what is being tested here and make sure that it is described well. Are we really testing "the services schema" or the `Instance` model?